src/client/datascience/ipywidgets/commonMessageCoordinator.ts

```diff
@@ -114,14 +114,10 @@ export class CommonMessageCoordinator {
         ]);
     }
 
-    private hash(s: string): string {
-        return this.hashFn().update(s).digest('hex');
-    }
-
     private sendLoadSucceededTelemetry(payload: LoadIPyWidgetClassLoadAction) {
         try {
             sendTelemetryEvent(Telemetry.IPyWidgetLoadSuccess, 0, {
-                moduleHash: this.hash(payload.moduleName),
+                moduleHash: getTelemetrySafeHashedString(payload.moduleName),
```

Comment:
Common function instead of genrating this by hand different way (the hash is still the same, hence there's no change to any of the old data)